src/mame/mame.lst

```diff
@@ -581,6 +581,9 @@ altos486
 @source:altos/altos8600.cpp
 altos8600
 
+@source:altos/altos586.cpp
+altos586
+
 @source:amiga/alg.cpp
```

Comment:
Please keep this file in order – this should come after the `altos/altos486.cpp` section.